function.go

```diff
@@ -46,3 +46,11 @@ func (fn *Function) NewInstance(args ...Valuer) (*Object, error) {
 	fn.ctx.deregister()
 	return getObject(fn.ctx, rtn), getError(rtn)
 }
+
+// Return the script origin for a function
+func (fn *Function) SourceMapUrl() (*Value, error) {
+	fn.ctx.register()
+	rtn := C.FunctionSourceMapUrl(fn.ptr)
+	fn.ctx.deregister()
```

Comment:
It doesn't look like we need to lookup the v8go.Context from the v8::Context in `FunctionSourceMapUrl`, so I don't think the register and deregister is necessary.  I think it is only necessary in the above functions because JS code is being called that could then call a Go callback.

```suggestion
	rtn := C.FunctionSourceMapUrl(fn.ptr)
```